server/events/commit_status_updater.go

```diff
@@ -34,9 +34,6 @@ type CommitStatusUpdater interface {
 	// UpdateCombinedCount updates the combined status to reflect the
 	// numSuccess out of numTotal.
 	UpdateCombinedCount(repo models.Repo, pull models.PullRequest, status models.CommitStatus, cmdName command.Name, numSuccess int, numTotal int) error
-	// UpdateProject sets the commit status for the project represented by
-	// ctx.
-	UpdateProject(ctx command.ProjectContext, cmdName command.Name, status models.CommitStatus, url string) error
```

Comment:
`type StatusUpdater interface` has same impl and this is used only for test. so removed.